api/chat_main_handler.go

```diff
@@ -61,8 +60,8 @@ type ChatCompletionResponse struct {
 
 type Choice struct {
 	Message      openai.ChatCompletionMessage `json:"message"`
-	FinishReason interface{} `json:"finish_reason"`
-	Index        int         `json:"index"`
+	FinishReason interface{}                  `json:"finish_reason"`
+	Index        int                          `json:"index"`
 }
 
 type Message struct {
```

Comment:
:robot: OpenAI

This patch only changes the formatting of the `Choice` struct. 

The fields `FinishReason` and `Index` were already present in the struct, so this change does not introduce any new functionality or bug risks.

Please confirm that the formatting change is intentional before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->